Feliz.Recharts/Legend.fs

```diff
@@ -0,0 +1,94 @@
+namespace Feliz.Recharts
+
+open System
+open Feliz
+open Fable.Core
+open Fable.Core.JsInterop
+
+[<Erase>]
+type legend =
+    static member inline width (value: int) = Interop.mkLegendAttr "width" value
+    static member inline height (value: int) = Interop.mkLegendAttr "height" value
+    static member inline iconSize (value: int) = Interop.mkLegendAttr "iconSize" value
+    static member inline chartWidth (value: int) = Interop.mkLegendAttr "chartWidth" value
+    static member inline chartHeight (value: int) = Interop.mkLegendAttr "chartHeight" value
+    static member inline margin (?top: int, ?left: int, ?right: int, ?bottom: int) = 
+        let margin = 
+            createObj [ 
+                "top" ==> Option.defaultValue 0 top
+                "left" ==> Option.defaultValue 0 left
+                "right" ==> Option.defaultValue 0 right
+                "bottom" ==> Option.defaultValue 0 bottom ]
+
+        Interop.mkLegendAttr "margin" margin
+    static member inline formatter (f: string -> 'a -> int) = Interop.mkLegendAttr "formatter" (System.Func<_,_,_> f)
```

Comment:
Why is the lambda type `string -> 'a -> int` what is `formatter` supposed to do?

Reply:
Here's an example in the Recharts docs. It's a function that can change how a particular legend is formatted. 
![image](https://user-images.githubusercontent.com/83406605/210256256-ebe513a6-ab5a-4eae-abba-0bd2add16af6.png)
![image](https://user-images.githubusercontent.com/83406605/210256287-313e9294-f1f1-4373-884e-e66af8dd16c3.png)
![image](https://user-images.githubusercontent.com/83406605/210256384-e31aabf9-7fdf-4b1d-8836-36745fd4a78c.png)

Reply:
According to this example, the type of the input function should be `string -> 'a -> ReactElement`, no?

Reply:
The int is the index, according to the docs. 

![Screenshot_20230102_163018](https://user-images.githubusercontent.com/83406605/210282103-d6e6d5af-abb7-42a4-9286-ca2fdd62517f.jpg)

Reply:
if that is the full type, then the lambda should be `string -> 'a -> int -> ReactElement`

Reply:
Ok got it, fixed!